apollo-router/src/router_factory.rs

```diff
@@ -198,6 +228,33 @@ impl YamlRouterFactory {
         // Process the plugins.
         let plugins = create_plugins(&configuration, &schema, extra_plugins).await?;
 
+        let tls_root_store = configuration
+            .tls
+            .subgraphs
+            .as_ref()
+            .and_then(|subgraphs| subgraphs.certificate_authorities.as_ref())
+            .and_then(|certificate_authorities| {
+                let mut store = RootCertStore::empty();
+                let certificates = load_certs(certificate_authorities)
```

Comment:
Is this chunk the same as above? Should it be a function or method, maybe on the `Subgraph` type?

Reply:
fixed in [5b0e6d6](https://github.com/apollographql/router/pull/2008/commits/5b0e6d61b2e91f73bb285e5eff57a9aa411e28d4)